### ISSUE TITLE
[1.19.2 IC2C 2.0.7.1] Nuclear reactors dumping items when reload the chunk

### ISSUE THREAD

**Issue by Lee8950** [TinyModularThings/IC2Classic#1201]
Reactor dumps heat vents, balancer, rods out when reloading chunk. The presented screenshot is already the third time the situation happens in a day. Server-sided mods and client-sided mods are listed below. Filtered extraction tubes and request tubes are connected to the reactors.
![Screenshot 2024-09-17 211450](https://github.com/user-attachments/assets/e6c45dcb-7fef-4619-bab9-28f498f2d4b5)
![Screenshot 2024-09-17 211316](https://github.com/user-attachments/assets/17aef9f5-e222-4a72-b313-b38490881dfd)
![Screenshot 2024-09-17 211436](https://github.com/user-attachments/assets/4c2989c1-7993-4b2e-9139-a6c1ced84539)

**Comment by Speiger** [comment]
o.O could you try to provide a testable scenario?
At best would be without other mods, but first get it going with your modpack.

Because you are the first person EVER to report this kind of issue.
(Note it will be fixed but i need your help to fix it)

**Comment by Speiger** [comment]
@Lee8950 it stays open. I will try it myself and at least have an honest look if there is a case.

Anyways it stays open until the next patch is out.

**Comment by Lee8950** [comment]
Update: After shutting down the server I copy the server files to my computer.
![2024-09-18_02 36 16](https://github.com/user-attachments/assets/aefc4173-24b9-40b3-8f7f-b6b0366a70bd)
![2024-09-18_02 36 22](https://github.com/user-attachments/assets/05a730c0-56dd-4bf5-a2d8-532db01e7d54)
![2024-09-18_02 36 24](https://github.com/user-attachments/assets/36910c96-7f47-46a5-83e9-144c7d05c5d2)
![2024-09-18_02 36 41](https://github.com/user-attachments/assets/97202a6e-8ebf-4111-a147-7a0ad0f0e7bc)
![2024-09-18_02 37 00](https://github.com/user-attachments/assets/3fd6af34-dab6-45d6-9a08-d168035805f4)
![2024-09-18_02 37 01](https://github.com/user-attachments/assets/db2e25ed-3a61-4cac-a249-f6212684bbdb)
![2024-09-18_02 36 11](https://github.com/user-attachments/assets/e1b86736-0ccf-4f1d-a834-731cfcb14624)
So basically you need to build a reactor like this. After setting the provider/requester tube, having clicked disconnect and reconnect then items drop.
Mod list(server/client)
![modlist](https://github.com/user-attachments/assets/7f3f5175-a483-47c2-8ff8-984dce977270)
![localmodlist](https://github.com/user-attachments/assets/7baad74e-3340-4947-9729-bd19d72b6d36)

To reproduce this, you might need to start a local server(i tries to reproduce locally but didn't work). We are running a forge-1.19.2-43.3.2 server.

Personally I think there is some issues about interaction between reactor and tube(provider/requester tube in this case), hope it might help you.

**Comment by Speiger** [comment]
@Lee8950 I have tried to reproduce this and have extreme difficulty. Especially since there is very little time i can investigate into this.
But i have fixed bugs in this coming patch that might be related in an indirect way, lets hope it works :)
